01/mock-interview-1-submission.md

```diff
@@ -0,0 +1,83 @@
+# Mock Interview #1 Retrospective
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: short-answer
+* id: 1f5bb96e-faa3-428f-b4fe-e1c374d8c960
+* title: Replit Link
+* topics: interview
+* points: 1
+##### !question
+
+Please enter a link to the Replit you used to complete your interview.
+
+##### !end-question
+##### !answer
+
+/.*/
+
+##### !end-answer
+### !end-challenge
+
+<!-- Question 2 -->
+### !challenge
+* type: short-answer
+* id: 2ed2b435-15b6-46cf-b3c2-e9762eea53b9
+* title: Mock Interview Partner
+* topics: interview
+##### !question
+
+What is the first and last name of the person you partnered with for your mock interview? If the individual was not another Adie intern and not part of the instructional staff, please list their association to you (e.g. partner, friend, mentor, intern manager).
```

Comment:
@kyra-at-ada do you think the addition of the person's association is worthwhile?

Reply:
Mmmm, I'm thinking about this and while I do want there to be flexibility, I'm hesitant to let them do their interviews with anyone or any question to fulfill this requirement. I'm open to discussion, but as I understand it, part of the intent behind this is to get them practice with the topics we're covering as part of CS Fundamentals. If they're using outside questions, it's more work on our part to categorize them and they may be out of scope questions, which isn't necessarily a bad thing but doesn't give them practice on the topics we want them to get practice in or might be too difficult and discouraging to the interns (ex. i've have interns come to me upset because they were trying to solve problems on currency arbitrage, distributed systems, etc. before they had a good grasp on the basics of what we do cover). I also really do think it's valuable for them to be the interviewer wherever they can which I'm not sure is going to happen if they're doing it with intern managers or mentor.

My recommendation would be to say they need to do 5 interviews with the Ada questions and that we encourage them to do additional interviews with other folks.

Reply:
That's totally understandable. I have a slightly differing thought pattern so let's discuss our thoughts a bit more offline 👍🏾